www/source/blog/2018-05-21-On-Prem-Builder.html.md

```diff
@@ -0,0 +1,45 @@
+---
+title: Introducing Habitat On-Premise Builder
+date: 2018-05-21
+author: Salim Alam
+tags: habitat, builder, depot, on-premise
+category: builder
+classes: body-article
+---
+
+### Introduction
+
+One of the frequent requests that we have had since we introduced the SaaS Habitat Builder last year has been to allow Habitat users to have the same functionality in their on-premise infrastructure. There are several reasons why this can be desirable - for example, having an enterprise policy that prohibits hosting of internal software in the public cloud, or having a completely "air-gapped" network isolation requirement. 
+
+Running Builder on-premise represents an interesting challenge. On the one hand, it is a fairly sophisticated set of scaled out services with a broad range of operational parameters. On the other hand, it is also entirely self-hosted with Habitat itself, and therefore can take advantage of all the Habitat capabilities for deploying, configuring and managing services. So the undertaking was not as daunting as it might have seemed otherwise.
+
+Earlier in the year we rolled out an alpha version of the on-premise Builder Depot to some of Chef's early-adopter partners. We are now happy to make the functionality available more broadly, so that more members of the Habitat community can make use of these capabilities.
+
+### What is Habitat Builder Depot?
+
+The Habitat Builder Depot is the set of Builder services that provide the capability to host Habitat packages. It comes with a Web front end to allow users to log in with a number of different authentication providers, and also hosts a REST API that Habitat clients (i.e., the `hab` binary running from any workstation) can be pointed to.
```

Comment:
I might remove the second occurrence of "Builder" from this sentence, so it reads `is the set of services that provide...`